src/photosui.cs

```diff
@@ -482,15 +485,18 @@ interface PHProjectTypeDescriptionInvalidator
 	[NoMac][NoTV]
 	[DisableDefaultCtor]
 	[NoMacCatalyst]
+#if NET // Can't apply Deprecated and Obsoleted to same element
 	[Deprecated (PlatformName.iOS, 13, 0)]
+#endif
+	[Obsoleted (PlatformName.iOS, 14, 0)] // Removed from headers completely
```

Comment:
I though we converted the Obsoleted to Deprecated in the generator, don't we?

Reply:
We do, and then you get 2 copies of the attribute, one which over writes the second (whoever comes last). I have a test explicitly to detect this.